app/views/shared/_footer.html.erb

```diff
@@ -103,23 +103,23 @@
             <div class="footer--icons">
                 <a class="facebook-logo" href=
                 "https://www.facebook.com/PULibrary" aria-label=
-                "Opens Princeton University Library Facebook page">
+                "PUL Facebook">
                 <%= image_tag('facebook.png', alt: "Facebook logo") %></a>
                 <a class="twitter-logo" href=
                 "https://twitter.com/PULibrary" aria-label=
-                "Opens Princeton University Library Twitter page">
+                "PUL Twitter">
```

Comment:
The screen reader reads PUL which can be confusing for a screen reader user. Can you please rename PUL to Princeton University Library ?

Reply:
Done.